doc/source/devel/howto_recorders.rst

```diff
@@ -48,7 +48,34 @@ the environment variables.
 Writing a custom recorder
 -------------------------
 
-.. todo:: document how to write custom recorders
+This chapter (still under development) provides the necessary information to
+write recorders in Sardana.
+
+Before writing a new recorder you should check the `Sardana plugins
+catalogue <https://github.com/sardana-org/sardana-plugins>`_.
+There's a chance that somebody already wrote one that fits to your needs.
+
+If finally you decide to write a new one below you can find some useful
+information:
+
+* Your recorder class would need to inherit from ``BaseFileRecorder`` and
```

Comment:
Shouldn't it inherit from `DataRecorder` instead of `BaseFileRecorder`?

For example, the `ShmRecorder` and the `OutputRecorder` inherit from `DataRecorder`

Reply:
You are right! I put two options in the docs: if your destination is a file use BaseFileRecorder if not use DataRecorder. Thanks!